frontend-landing/assets/css/team_style.css

```diff
@@ -0,0 +1,276 @@
+.team-intro {
+  position: relative;
+}
+
+.explore video {
+  object-fit: cover;
+}
+
+.team-intro-container {
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  position: absolute;
+  background: url(../../assets/backgrounds/cyan-gradient.png) top -900px right -900px no-repeat, linear-gradient(180deg, rgba(4, 13, 77, 0.7) -43.4%, rgba(50, 3, 111, 0.7) 222.51%);
+  align-items: center;
+}
+
+.team-intro-title {
+  color: #F5FCFE;
+  line-height: 100%;
+}
+
+.team-intro-title>span {
+  color: #73D6F1;
+}
+
+.team-intro-content {
+  width: 100%;
+  margin-top: 30px;
+  line-height: 25px;
+}
+
+.team-member-bg {
+  padding: 100px 0;
+  background: url(../../assets/backgrounds/cyan-gradient.png) top -500px left -1000px no-repeat,
+    url(../../assets/backgrounds/cyan-gradient.png) bottom -300px right -1000px no-repeat;
+}
+
+.team_card-alignStretch {
+  display: flex;
+  flex-flow: column;
+  align-items: stretch;
+}
+
+.team_card-bg {
+  height: 200px;
+  padding: 90px 20px 50px;
+  position: relative;
+  background: #1D277A;
+  margin-top: 80px;
+  text-align: center;
+  border-radius: 10px;
+}
+
+.team_card-img {
+  top: -10px;
+  left: 50%;
+  position: absolute;
+  transform: translate(-50%, -50%);
+}
+
+.team_card-btn {
+  position: relative;
+  line-height: 0;
+}
+
+.MuiBox-root-54 {
+  display: inline-block;
+  position: relative;
+  line-height: 0;
+}
+
+.makeStyles-root-51 {
+  cursor: pointer;
+  display: inline-block;
+  z-index: 102;
+  position: relative;
+  line-height: 0px;
+}
+
+.team_card-photo {
+  top: 50%;
+  left: 50%;
+  width: 115px;
+  filter: grayscale(1);
+  display: inline-flex;
+  position: absolute;
+  transform: translate(-50%, -50%);
+  align-items: center;
+  border-radius: 50%;
+  justify-content: center;
+}
+
+.team_card-name {
+  color: #F5FCFE;
+  font-size: 24px;
+  font-style: normal;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: 900;
+  line-height: 110%;
+  margin-bottom: 5px;
+}
+
+.team_card-role {
+  color: #73D6F1;
+  font-size: 18px;
+  font-style: normal;
+  font-family: "Museo Sans", sans-serif;
+  font-weight: 600;
+  line-height: 110%;
+  margin-bottom: 2px;
+}
+
+.team_card-info {
+  color: #F5FCFE;
+  opacity: 1;
+  font-size: 16px;
+  font-style: normal;
+  font-family: "Museo Sans", sans-serif;
+  line-height: 112%;
+  white-space: pre-line;
+}
+
+.team_card-info>strong {
+  font-weight: 900;
+}
+
+.team_card-socials {
```

Comment:
This CSS looks janky. Can we find a better way to center... such as `display: flex; margin: auto`... and remove the `50%`s and `transform`?